.env

```diff
@@ -1,3 +1,10 @@
 # Local Dev Environment
 VITE_APP_NAME=PURIS
-VITE_BASE_URL=http://localhost:8081/catena
\ No newline at end of file
+VITE_BACKEND_BASE_URL=http://localhost:8081/catena/
+VITE_ENDPOINT_MATERIALS=stockView/materials
+VITE_ENDPOINT_PRODUCTS=stockView/products
+VITE_ENDPOINT_MATERIAL_STOCKS=stockView/material-stocks
+VITE_ENDPOINT_PRODUCT_STOCKS=stockView/product-stocks
+VITE_ENDPOINT_CUSTOMER=stockView/customer?materialUuid=
+VITE_ENDPOINT_PARTNER_PRODUCT_STOCKS=stockView/partner-product-stocks
+VITE_ENDPOINT_UPDATE_PARTNER_PRODUCT_STOCK=stockView/update-partner-product-stock?materialUuid=
```

Comment:
I suggest ending every file with an empty newline.

Reply:
should be fixed with commit d7e914d251ec30214ef905cfd0e2a112471dd13b